spec/unit/event_dispatch/dispatcher_spec.rb

```diff
@@ -123,7 +123,7 @@ def synchronized_cookbook(cookbook_name, cookbook)
   context "events that queue events" do
     class Accumulator
       def self.sequence
-        @secuence ||= []
+        @sequence ||= []
```

Comment:
This was the bug I think?

Reply:
Nothing accesses that variable outside this function so it probably wasn't breaking anything.